tests/select_for_update/tests.py

```diff
@@ -179,6 +180,19 @@ def test_for_update_requires_transaction_only_in_execution(self):
         with self.assertRaises(transaction.TransactionManagementError):
             list(people)
 
+    @skipUnlessDBFeature('supports_select_for_update_with_limit')
+    def test_select_for_update_with_limit(self):
+        other = Person.objects.create(name='Grappeli')
+        with transaction.atomic():
+            qs = list(Person.objects.all().order_by('pk').select_for_update()[1:2])
```

Comment:
I would also add a comment explaining why we use `list()` and `[0]` rather than `get`, but it's also ok without it.